app/assets/stylesheets/dm/pages/_facilities.scss

```diff
@@ -28,6 +28,10 @@
       }
     }
 
+    .fa-bookmark:hover{
```

Comment:
we probably want a treatment similar to the `dm-practice-bookmark-btn` class in the `_practice_card.scss` file

```
    .fas {
      color: color($theme-color-primary-vivid);

      &:hover {
        color: color($theme-color-primary-dark);
      }

      &:active {
        color: color($theme-color-primary-darker);
      }
    }

    .far {
      color: color($theme-color-base-ink);

      &:hover {
        color: color($theme-color-primary-vivid);
      }

      &:active {
        color: color($theme-color-primary-dark);
      }
    }
 ```

Reply:
Yeah - that works great ^^ Thanks Elisa!